ParkingLotApi/Repositories/ParkingLotsRepository.cs

```diff
@@ -37,5 +38,18 @@ public async Task<ParkingLot> GetParkingLotById(string id)
         {
             return await _parkingLotCollection.Find(e => e.Id == id).FirstAsync();
         }
+
+        public async Task<ParkingLot> PutParkingLot(string id, int capacity)
```

Comment:
update the capacity in object can write into Service level as it is a business logic.